packages/treasury/test/test-bootstrapPayment.js

```diff
@@ -28,7 +34,7 @@ const makeInstall = async (root, zoe) => {
 
 test('bootstrap payment', async t => {
   const zoe = makeZoe(fakeVatAdmin);
-  const autoswapInstall = await makeInstall(autoswapRoot, zoe);
+  const autoswapInstall = await makeInstall(await autoswapRoot, zoe);
```

Comment:
This is  a non-top-level await. Is that going to be allowed? Can we put the await inside `makeInstall()` instead?